docs/release-notes/8.8.asciidoc

```diff
@@ -34,10 +66,8 @@ There are no breaking changes in 8.8.1.
 [discrete]
 [[bug-fixes-8.8.1]]
 ==== Bug fixes 
-* Fixes bugs in the Anomalies table that left {ml} jobs greyed out after they were installed ({pull}158821[#158821]), and that stopped some job counts from being shown ({pull}158739[#158739]).
 * Fixes a bug that caused all field types in the **Fields** browser to appear as `unknown` ({pull}158594[#158594]).
 * Fixes a bug that caused the **Add rule exception** flyout to load indefinitely when index fields couldn't be retrieved ({pull}158371[#158371]).
-* Allows actions to be taken on fields with multiple values ({pull}158060[#158060]).
```

Comment:
@machadoum would you mind confirming that this was only fixed in 8.8.2 and _not_ 8.8.1? I removed this summary after noticing [this comment](https://github.com/elastic/kibana/pull/158060#issuecomment-1583069201) about the fix not making it into the latest BC for 8.8.1.

Reply:
Hey! You can trust the comment. It didn't make it for 8.8.1.